great_expectations_cloud/agent/agent.py

```diff
@@ -134,17 +135,17 @@ class GXAgent:
     def __init__(self: Self):
         agent_version: str = self.get_current_gx_agent_version()
         great_expectations_version: str = self._get_current_great_expectations_version()
-        print(f"GX Agent version: {agent_version}")
-        print(f"Great Expectations version: {great_expectations_version}")
-        print("Initializing the GX Agent.")
+        LOGGER.info(f"GX Agent version: {agent_version}")
```

Comment:
nit, can we put dynamic values in `extra`?

Reply:
Actually do we need 3 messages here? Could we do just one `Initializing the GX Agent` message with `agent_version` and `gx_core_version` in `extra`?